### ISSUE TITLE
Python 3?

### ISSUE THREAD

**Issue by waveform80** [waveform80/compoundpi#37]
Now that python-daemon is *finally* py3 compatible should we move stuff over? It'd certainly clean up the command line client a whole lot (Cmd is an old-style class in py2 which makes sub-classing it a pain, and then there's the whole argparse thing). Not a definite thing, but something to consider sometime prior to 1.0.

**Comment by mananwason** [comment]
Hi @waveform80. Have you switched over to Python 3 as of now?

**Comment by waveform80** [comment]
I won't be moving the project completely to Python 3 (i.e. I'd like the client library to be accessible from both Python 2 and 3 for the time being), but there are a whole bunch of commits in the recent history for 3.2+ compatibility: 2dfaedae0b71e5ddc27d68cad81d0ad44d334ed6, e4482b60a7e4d66c70e94a815f0bd5c6e8a2360c, 6b1125558950819f7e0f566ff83ab111f317dd8f, etc. and I *think* the master branch is now Python 3.2+ compatible.

At least, the [current test suite passes](https://travis-ci.org/waveform80/compoundpi) on 3.2+ (though I'll admit the test suite is still far from comprehensive; ignore the coverage stats at the end there - they're only for the modules I'm currently testing which basically excludes everything in the CLI and GUI interfaces!).

In other words, you should be safe using the client library on Python 3 but I make no guarantees about the CLI or GUI interfaces yet!
